gapis/api/vulkan/api/descriptor.api

```diff
@@ -899,6 +905,7 @@ cmd void vkCmdPushConstants(
     VkShaderStageFlags stageFlags,
     u32                offset,
     u32                size,
+    @len("size")
```

Comment:
Same as above, we probably want a `@len_bytes` or similar

Reply:
Done